src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.UnitConverter/Community.PowerToys.Run.Plugin.UnitConverter.csproj

```diff
@@ -11,19 +11,17 @@
     <useWPF>true</useWPF>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
-    <Platforms>x64</Platforms>
+    <OutputPath>..\..\..\..\$(Platform)\$(Configuration)\modules\launcher\Plugins\UnitConverter\</OutputPath>
```

Comment:
Comparing the diff, I think your missing a `..` here.

Reply:
You're correct. I'll get these fixed. Those `..` tripped me :D